src/k8sdeploy/k8sdeploy.py

```diff
@@ -196,4 +202,48 @@ def deploy_objects(self,action="apply",delete_namespace=False):
             if self.rollout_restart:
                 self.deployment_rollout_restart()
         elif self.vars['deploy_type'].lower() in ['job','cronjob']:
-            self.load_deploy("jobs",action)
\ No newline at end of file
+            self.load_deploy("jobs",action)
+
+class EksUpateConfig():
+    def __init__(self,stack: str,github_runner_ip: str):
+        self.stack=stack
+        #self.action=action
+        self.cluster_name=f"adexk8s-eks-cluster-{self.stack}"
+        self.ip4 = github_runner_ip
+        self.eks=boto3.client('eks')
+
+    @contextmanager
+    def disable_exception_traceback(self):
+        """
+        All traceback information is suppressed and only the exception type and value are printed
+        """
+        default_value = getattr(sys, "tracebacklimit", 1000)  # `1000` is a Python's default value
```

Comment:
what does this default value mean, practically speaking?

Reply:
default_value is the length of the trackeback. I use this when a command-line tool: removes traceback and gives functional information to the user of the tool.